README.md

```diff
@@ -1,5 +1,7 @@
 # Togliatti, Russian Federation
 
+<img src="https://github.com/beerjs/togliatti/blob/edit-readme/beerjs-tlt.png?raw=true" alt="BeerJS Еogliatti" width="600" height="444" />
```

Comment:
[BeerJS Togliatti](https://beerjs.github.com/togliatti/blob/edit-readme/beerjs-tlt.png)


🤣 Have you forgotten the Markdown syntax?